packages/aws-cdk/lib/api/garbage-collection/garbage-collector.ts

```diff
@@ -536,11 +535,11 @@ export class GarbageCollector {
         });
 
         const deletedCount = batch.length;
-        await this.ioHelper.notify(debug(`Deleted ${deletedCount} assets`));
+        await this.ioHelper.notify(IO.DEFAULT_TOOLKIT_DEBUG.msg(`Deleted ${deletedCount} assets`));
         printer.reportDeletedAsset(deletables.slice(0, deletedCount));
       }
     } catch (err) {
-      await this.ioHelper.notify(info(chalk.red(`Error deleting images: ${err}`)));
```

Comment:
Intentional color removal?

Reply:
Intentional. Turned the red info message that says "Error" into a proper error level message, which will be colored red as well.